apps/cow-fi/components/TokensList/index.tsx

```
@@ -14,6 +14,13 @@ import {
   NoTokensText,
 } from './index.style'
 import { clickOnToken } from 'modules/analytics'
+import type { CowProtocolTheme } from 'styled-components'
```

ESLint

[error] 'styled-components' import is restricted from being used. Please import from styled-components/macro.

(no-restricted-imports)